tests/project-tests/test_projects.sh

```diff
@@ -30,15 +30,8 @@ while read project_name branch ; do
 
     podir="$(mktemp -d ${project_name}-${branch}.XXXXXX)"
 
-    # The zanata output only matters if something goes wrong
-    zanata_output="$(zanata pull --project-type gettext --project-id "$project_name" --project-version "$branch" --transdir "$podir" --url https://fedora.zanata.org/ 2>&1)"
-    if [ $? -ne 0 ]; then
-        echo "Zanata pull failed for $project_name:$branch"
-        echo "$zanata_output"
-        status=1
-        rm -rf "$podir"
-        continue
-    fi
+    # Download translations
+    git clone --depth 1 -b $branch -- "https://github.com/rhinstaller/anaconda-l10n.git" $podir
```

Comment:
This should not be hard-coded to Anaconda git. Let's use ``$project_name`` or change this other way around.

Reply:
Ok, I've changed this to run tests on the `rhinstaller/*-l10n` repos. I'm not really sure what was the intention behind this test/make target, it' not really a test case for translation canary, it looks more like a helper target to run translation canary against all rhinstaller projects. The test itself is now failing, but the errors look like legit errors in the PO files for anaconda and python-meh so I assume it works as expected.